pkg/api/handlers/libpod/images.go

```diff
@@ -44,12 +44,14 @@ func ImageExists(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
 	name := utils.GetName(r)
 
-	_, err := runtime.ImageRuntime().NewFromLocal(name)
+	ir := abi.ImageEngine{Libpod: runtime}
+	report, err := ir.Exists(r.Context(), name)
 	if err != nil {
 		utils.Error(w, "Something went wrong.", http.StatusNotFound, errors.Wrapf(err, "failed to find image %s", name))
 		return
 	}
-	utils.WriteResponse(w, http.StatusNoContent, "")
+
+	utils.WriteResponse(w, http.StatusOK, report)
```

Comment:
This looks like a breaking change to the API

Reply:
I don't know, I believe the previous version was broken.  @jwhonce WDYT?

Reply:
Ok I went back to original API, although I still believe it is slightly broken, but usable.